Program.cs

```diff
@@ -4,17 +4,11 @@
 string userinput = Console.ReadLine ();
 string input = userinput.ToLower ();
 List<char> order = new List<char> ();
-for (int i = 0; i < input.Length; i++) {
-   if (input[i] != ' ') {
-      order.Add (input[i]);
-   }
-}
+for (int i = 0; i < input.Length; i++) if (input[i] != ' ') order.Add (input[i]);
 List<char> reverseorder = Enumerable.Reverse (order).ToList ();

```

Comment:
Rewrite by traversing in the given string no need to get the reverse of whole string and do a compare.